scripts/upgrade

```diff
@@ -191,6 +191,12 @@ do
     # Create a temporary directory
     tmpdir=$(mktemp -d)
 
+    # Patch nexcloud files only for the last upgrade.
+    if [ "$current_major_version" = "last" ]
+    then
+        cp -a ../sources/patches_last_version ../sources/patches
```

Comment:
Wouldn't it be cleaner if you added that line and the corresponding comment on L179...?

Reply:
Hum, yes... Why I didn't do that ...?